### ISSUE TITLE
Hello author, will this repository still update its code

### ISSUE THREAD

**Issue by Miraitwo** [Yuan-Feng47/Window-Attention-Convolution-Network#1]

**Comment by Yuan-Feng47** [comment]
Yes, the paper and code will be uploaded to github during China's National Day holiday. I used to work overtime and travel a lot. I was so busy.




---Original---
From: ***@***.***&gt;
Date: Thu, Sep 26, 2024 11:12 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [Yuan-Feng47/Window-Attention-Convolution-Network] Hello author, willthis repository still update its code (Issue #1)

**Comment by Miraitwo** [comment]
Thank you very much for your reply
